addons/mail/static/src/components/call_sidebar_view/call_sidebar_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+<t t-name="mail.CallSidebarView" owl="1">
+    <t t-if="callSidebarView">
+        <div class="o_CallSidebarView d-flex align-items-center h-100 flex-column">
+            <t t-foreach="callSidebarView.callView.tileParticipantCards" t-as="participantCard" t-key="'grid_tile_'+participantCard.localId">
```

Comment:
I'm relatively confident that the hack with `'grid_tile'` is no longer needed.

It was necessary because there were 2 use-cases of `ParticipantCard` in `CallView`.
But now that they are isolated in different components, we no longer need to help OWL reconciliation with `t-key` in same context (context being same component)